app/src/hooks/api/useObservationApi.ts

```diff
@@ -60,9 +72,42 @@ const useObservationApi = (axios: AxiosInstance) => {
     return data;
   };
 
+  /**
+   * Get observation submission based on survey ID
+   *
+   * @param {number} projectId
+   * @param {number} surveyId
+   * @returns {*} {Promise<IGetObservationSubmissionResponse>}
+   */
+  const getObservationSubmission = async (
+    projectId: number,
+    surveyId: number
+  ): Promise<IGetObservationSubmissionResponse> => {
+    const { data } = await axios.get(`/api/project/${projectId}/survey/${surveyId}/observation/submission/get`);
+
+    return data;
+  };
+
+  /**
+   * Initiate the validation process for the submitted observations
+   * @param {number} submissionId
+   */
+  const initiateObservationSubmissionValidation = async (submissionId: number) => {
+    axios.post(`/api/dwc/validate`, {
+      occurrence_submission_id: submissionId
+    });
+  };
+
+  const initiateXLXvalidation = async (submissionId: number) => {
```

Comment:
Dont think we need this function - just pass in the file type to the `initiateObservationSubmissionValidation` function and make the respective call from there

Reply:
and if we really do want this function, then it needs to be renamed to `initiateXLSXValidation`